source/robot_model/include/robot_model/Model.hpp

```diff
@@ -14,6 +14,7 @@
 #include <state_representation/space/Jacobian.hpp>
 #include <state_representation/space/joint/JointState.hpp>
 #include <state_representation/space/cartesian/CartesianState.hpp>
+#include "robot_model/QPSolver.hpp"
```

Comment:
nitpick (since we are already editing this part): in the style guide, we order imports by standard lib, external libs, internal libs and then local headers, which you have done correctly here. But it would be nice to separate the groups by a newline:

```c++
#include <string>
#include <vector>
#include <optional>

#include <OsqpEigen/OsqpEigen.h>
#include <pinocchio/algorithm/crba.hpp>
#include <pinocchio/algorithm/rnea.hpp>
#include <pinocchio/multibody/data.hpp>
#include <pinocchio/parsers/urdf.hpp>
#include <pinocchio/algorithm/geometry.hpp>

#include <state_representation/parameters/Parameter.hpp>
#include <state_representation/parameters/ParameterInterface.hpp>
#include <state_representation/space/Jacobian.hpp>
#include <state_representation/space/joint/JointState.hpp>
#include <state_representation/space/cartesian/CartesianState.hpp>

#include "robot_model/QPSolver.hpp"
```

Reply:
Nice! Yes makes sense, I have updated the imports.